src/features/connections/services/connectionsApiSlice.ts

```diff
@@ -1,2 +1,45 @@
-// boiler plate
-import React from "react";
\ No newline at end of file
+import {
+  ConnectionsModel,
+  ConnectionMutationResponseModel,
+} from "../models/Connections.model";
+import { apiSlice } from "../../api/apiSlice";
+
+export const connectionsApiSlice = apiSlice.injectEndpoints({
+  endpoints: (builder) => ({
+    getConnections: builder.query<ConnectionsModel, number>({
+      query: (userId: number) => ({
+        url: `/users/${userId}/connections`,
+        method: "GET",
+      }),
+    }),
+    sendConnection: builder.mutation<ConnectionMutationResponseModel, number>({
+      query: (userId: number) => ({
+        url: `/users/${userId}/connections`,
+        method: "POST",
+      }),
+    }),
+    acceptConnection: builder.mutation<ConnectionMutationResponseModel, number>(
+      {
+        query: (userId: number) => ({
```

Comment:
nit: you can remove the typing for the parameter since you specified it on line 21
<ConnectionMutationresponseModel, number>

Reply:
sure